test/README.md

```diff
@@ -122,8 +122,8 @@ deactivate
             .....
             .....
             .....
-    def test_binaries(self, binary, package, version, are_in, not_in):
-        self._binary_test(binary, package, version, are_in, not_in)
+    def test_binaries(self, binary, product, version, are_in, not_in):
```

Comment:
It is also module.

Reply:
So it is here too another Readme missed.